src/sentry/integrations/slack/action_endpoint.py

```diff
@@ -36,17 +42,23 @@ class SlackActionEndpoint(Endpoint):
     authentication_classes = ()
     permission_classes = ()
 
-    def api_error(self, error, action_type, logging_data):
+    def api_error(self, error, action_type, logging_data, text):
```

Comment:
@NisanthanNanthakumar I think it might more clear if this argument was something like `unauthorized_error_text` since we only use this text of a 403 error.